BitwardenShared/UI/Auth/IntroCarousel/IntroCarouselView.swift

```diff
@@ -0,0 +1,121 @@
+import SwiftUI
+
+// MARK: - IntroCarouselView
+
+/// A view that allows the user to swipe through the intro carousel and then proceed to creating an
+/// account or logging in.
+///
+struct IntroCarouselView: View {
+    // MARK: Properties
+
+    /// An environment variable for getting the vertical size class of the view.
+    @Environment(\.verticalSizeClass) var verticalSizeClass
+
+    /// The `Store` for this view.
+    @ObservedObject var store: Store<IntroCarouselState, IntroCarouselAction, Void>
+
+    /// The index of the currently visible page in the carousel.
+    @SwiftUI.State private var tabSelection = 0
+
+    // MARK: View
+
+    var body: some View {
+        VStack(spacing: 0) {
+            TabView(selection: $tabSelection.animation()) {
+                ForEachIndexed(store.state.pages) { index, page in
+                    GeometryReader { reader in
+                        dynamicStackView {
+                            pageView(page)
+                        }
+                        .padding(.vertical, 16)
+                        .frame(maxWidth: .infinity, minHeight: reader.size.height)
+                        .scrollView(addVerticalPadding: false)
```

Comment:
🤔 By putting the scroll view here, on landscape modes with long text or big text (due to font size), it means that the image scrolls alongside the text, rather than being separate from the text-as-scrollview. Is this what we want? (On the Authenticator, I split the image out in landscape so only the text scrolled; not sure what the proper choice actually is, though, but both apps should eventually match in that regard)

Reply:
Good callout! I will update this to just scroll the text.